onebusaway-android/src/main/res/layout/arrivals_list_item_style_b.xml

```diff
@@ -79,6 +79,21 @@
                             android:ellipsize="end"/>
 
                     <ImageButton
+                            android:id="@+id/route_discussion"
```

Comment:
I'd like to have the following order of buttons from left to right, to match the order in the options menu and importance to users:

* Star
* Map
* Join discussion

I'm wondering if we add the "trip status" button to the card as well, as we've gotten a lot of feedback from users that love this feature - so it would be:

* Star
* Map
* Trip status
* Join discussion

Could you please give this a shot and see how it looks to have 4 buttons, and post some screenshots?  I'm not sure if it will be too crowded...